src/main/java/racingcar/domain/Car.java

```diff
@@ -32,28 +25,29 @@ public int getWinNum() {
 		return winNum;
 	}
 
-	public void increaseWinNum() {
-		winNum++;
+	public boolean isSamePosition(int position) {
+		return this.position == position;
 	}
 
-	public int createRandomNum() {
-		return (int)(Math.random() * ZERO_TO_NINE_CONVERTER);
+	public void increaseWinNum() {
+		winNum++;
 	}
 
 	public void race(int num) {
 		int positionStandard = position;
 		for (int i = 0; i < num; i++) {
-			move();
+			move(RandomNum.createRandomNum());
 		}
 		position -= positionStandard;
-		Output.printRace(name, position);
 	}
 
-	public boolean move() {
-		if (createRandomNum() >= MOVE) {
+	private void move(int randomNum) {
+		if (randomNum >= MOVE) {
```

Comment:
car의 move가 random number 라는 것을 알 필요가 있을까요?
```suggestion
	private void move(int number) {
		if (number >= MOVE) {
```

Reply:
혹시 이 부분은 반영하신건가요?